drivers/go-getting-started.md

```diff
@@ -31,7 +83,7 @@ running on localhost.
 import (
 	"fmt"
 
-	driver "github.com/arangodb/go-driver"
+	A "github.com/arangodb/go-driver"
```

Comment:
What's the purpose of changing this alias to uppercase `A` ?

Reply:
IMHO this should stay a `driver` alias or remove an alias at all (it's still going to be driver)